supporting_content/dimensions/resourceid.md

```diff
@@ -0,0 +1,55 @@
+| GCP       | Big Query Billing Export | resource.name? resource.global_name? |
+| Microsoft | Cost details             | ResourceId                           |
+
+## Examples
+| Provider  | Scenario                                          | Pattern                                                                                          |
+|-----------|---------------------------------------------------|--------------------------------------------------------------------------------------------------|
+| AWS       | ARN provided which includes a resource name or id | arn:partition:service:region:account-id:resource-id <br>arn:partition:service:region:account-id:resource-type/resource-id<br> arn:partition:service:region:account-id:resource-type:resource-id|
+| AWS       | Some cases, just a resource-id is provided        | I-0b21f4c4434558933 OR my-cur-bucket                                                             |
+| AWS       | Tax, API call charges etc.                        | (null)                                                                                           |
+| GCP       | Resource based                                    | //container.googleapis.com/projects/<project_id>/locations/<location>/clusters/<cluster>         |
+| GCP       | Non-resource based (e.g. network egress)          | (null)                                                                                           |
+| Microsoft | Resource group resources                          | /subscriptions/<guid>/resourceGroups/<name>/providers/<provider>/<resource-type>/<resource-name> |
+| Microsoft | Subscription resources                            | /subscriptions/<guid>/providers/<provider>/<resource-type>/<resource-name>                       |
+| Microsoft | Tenant resources                                  | /providers/<provider>/<resource-type>/<resource-name>                                            |
+| Microsoft | Purchase charges                                  | (null)                                                                                           |
+| Microsoft | Reservation purchases                             | /providers/Microsoft.Capacity/reservationOrders/<guid>                                           |
+| Microsoft | Savings plan purchases                            | /providers/Microsoft.Capacity/savingsPlanOrders/<guid>                                           |
+
+### Documentation
+
+AWS:
+> <https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html>
+
+Interesting note on the "The resource identifier" which is at the end of the ARN: This is the name of the resource, the ID of the resource, or a [resource path](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html#arns-paths). Some resource identifiers include a parent resource (sub-resource-type/parent-resource/sub-resource) or a qualifier such as a version (resource-type:resource-name:qualifier)."
```

Comment:
Do we care to define ARN?

Reply:
I don't think so. This is really meant to capture scratch notes IMO. It's not going in the spec. I was considering not even converting to markdown to allow for it to be updated easily without needing to convert back-and-forth.

I've updated to indicate this is from the link above (which is official doc on ARN so should be sufficient)